src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -203,7 +231,11 @@ static int uct_rdmacm_iface_process_event(uct_rdmacm_iface_t *iface, struct rdma
     uct_rdmacm_ctx_t *cm_id_ctx;
     uct_rdmacm_ep_t *ep = NULL;
     ssize_t priv_data_ret;
-    int destroy_cm_id = 0;
+    ucs_status_t status;
```

Comment:
looks like 'status' variable is not really needed

Reply:
Why? It's needed to handle `RDMA_CM_EVENT_UNREACHABLE` when it should be `RDMA_CM_EVENT_REJECTED` but not supported for `RDMA_PS_UDP`